Runner/suites/Multimedia/Video/iris_v4l2_video_encode/run.sh

```diff
@@ -1,25 +1,40 @@
 #!/bin/sh
+
+# Copyright (c) Qualcomm Technologies, Inc. and/or its subsidiaries.
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+
 # Import test suite definitions
-/var/Runner/init_env
-TESTNAME="video_encode"
+# shellcheck source=/var/Runner/init_env
+. "${PWD}"/init_env
+TESTNAME="iris_v4l2_video_encode"
+TAR_URL="https://github.com/qualcomm-linux/qcom-linux-testkit/releases/download/IRIS-Video-Files-v1.0/video_clips_iris.tar.gz"
 
 #import test functions library
-. $TOOLS/functestlib.sh
+# shellcheck source=/var/Runner/utils/functestlib.sh
+. "${TOOLS}"/functestlib.sh
 test_path=$(find_test_case_by_name "$TESTNAME")
 log_info "-----------------------------------------------------------------------------------------"
 log_info "-------------------Starting $TESTNAME Testcase----------------------------"
 
 log_info "Checking if dependency binary is available"
 check_dependencies iris_v4l2_test
+check_network_status
```

Comment:
do you need to check network status here? I think it should go inside the conditional block where you download the file if it doesn't exist